Documentation/Storage-Configuration/.pages

```diff
@@ -0,0 +1,6 @@
+nav:
+    - ceph-dashboard.md
+    - ceph-monitoring.md
+    - key-management-system.md
+    - ceph-toolbox.md
+    - ...
```

Comment:
How about this order (? And almost everything could be in a folder, maybe just not the cleanup.
- Block Storage (RBD)
- Shared Filesystem (CephFS)
- Object Storage
- Monitoring --> Add the Ceph dashboard and Prometheus pages in this folder
- Ceph CSI
- Cleanup
- Advanced --> Add the KMS topic here